Wireframe/index.html

```diff
@@ -4,10 +4,56 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
+    <!--Header section-->
+    <header class="header-container">
+    <h1>What is Git</h1>
+    <p>Git is a free, open-source version control system that's used by developers to track and manage source code.</p>
+    </header>
+    <!--Main section-->
+    <main class="main-container">
+      <section class="main-section">
+        <!--First and middle section-->
+        <p>
+          <h2>Why developers use git?</h2>
+          Developers use Git, for a number of reasons, including: 
+<span>Collaboration</span>
```

Comment:
What semantic value does span carry?

Reply:
Hello, Sally. Thank you for your comment, and I'm glad to have you.  

I request PR without  "Need review" label. Then after I enabled it, the PR request format changed to default, when I tried to update it, it did not allow me. So I need help with how to update the PR request format.

 I removed span because it doesn't  do anything and will push the update.